tests/page-objects/default/contacts/contacts.wdio.page.js

```diff
@@ -363,11 +363,17 @@ const exportContacts = async () => {
   await (await exportButton()).click();
 };
 
-const getCardFieldInfo = async (label) => {
-  return {
-    label: await (await $(`.cell.${label} label`)).getText(),
```

Comment:
This never worked. We didn't have a class for the label, and even if we did, we we using spaces in the label which would have broken this CSS selector completely.